### PR TITLE
EDO-229 add reporter to training

### PR DESCRIPTION
**Pull request by 5Heinzelmann**

### CODE REVIEW

File: src/main/webapp/i18n/de/training.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
             "contactPerson": "Contact Person",
             "link": "Link",
             "validUntil": "Valid Until",
-            "validUntilPlaceholder": "z.B. 2029-01-01T00:00:00+01:00",
```
Comment (sberlin) [inline src/main/webapp/i18n/de/training.json]: The placeholder is important, but deleted by the generator here, I suppose.

File: src/main/webapp/i18n/en/training.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
             "contactPerson": "Contact Person",
             "link": "Link",
             "validUntil": "Valid Until",
-            "validUntilPlaceholder": "e.g. 2029-01-01T00:00:00+01:00",
```
Comment (sberlin) [inline src/main/webapp/i18n/en/training.json]: see https://github.com/iteratec/slab-dojo/pull/39/files#diff-4813a67b1ca9366cbcedf888a35e1ac2L23

Comment (5Heinzelmann) [reply src/main/webapp/i18n/en/training.json]: I added the translation value to both files (de and en)

File: src/main/webapp/i18n/de/training.json (hunk 1 above)
Comment (5Heinzelmann) [reply src/main/webapp/i18n/de/training.json]: see https://github.com/iteratec/slab-dojo/pull/39#discussion_r260258518

**Pull request merged**